src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -979,6 +979,34 @@ def list_permission_grants(client, identifier=None, query_filter=None, show_reso
     return result
 
 
+def app_federated_credential_list(client, identifier):
+    object_id = _resolve_application(client, identifier)
+    # This call is too simple, so it's unnecessary to extract a common method for both application and servicePrincipal
+    # and invoke it like
+    # _federated_identity_credential_list(client.application_federated_identity_credential_list, object_id)
+    return client.application_federated_identity_credential_list(object_id)
```

Comment:
We should be able to directly bind to the SDK and remove this custom function, but since we haven't decided whether to use flattened params or JSON (https://github.com/Azure/azure-cli/pull/22727#discussion_r888945820), we leave it here for future customization.